docs/modules/aerogear-push-chrome/pages/index.adoc

```diff
@@ -0,0 +1,10 @@
+The following step-by-step guides, gives you an introduction on how to
```

Comment:
Is this still relevant? At the moment UPS only supports Firebase (for Android push), APNS (Apple) and Windows Push. This seems to be about pushing to Chrome using GCM.